src/User/RegistrationForm.jsx

```diff
@@ -15,9 +16,31 @@ const RegistrationForm = () => {
     lastName: "",
     email: "",
     occupation: "",
+    managerId: "",
```

Comment:
If the Id is a Number the default state probably should be `null` and not an empty string